### PR TITLE
Add CI with build and python-publish workflows

### PR DESCRIPTION
**Pull request by lebrice**
Signed-off-by: Fabrice Normandin <fabrice.normandin@gmail.com>

### CODE REVIEW

File: .github/workflows/python-publish.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This workflows will upload a Python Package using Twine when a release is created
```
Comment (breuleux) [inline .github/workflows/python-publish.yml]: With Poetry, publishing to PyPI is `poetry publish --build`. But honestly I'd just leave out the publish workflow for now.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      if: ${{ matrix.python-version == '3.10' }}
+      run: poetry run black --check .
+
+    - name: Sort imports
+      if: ${{ matrix.python-version == '3.10' }}
+      run: poetry run isort -c --df .
+
+    - name: Test with pytest
+      run: poetry run pytest
+      # run: poetry run pytest --cov=ptera --cov-report term-missing
```
Comment (breuleux) [inline .github/workflows/build.yml]: Let's just remove the comments.

**Comment by breuleux** [comment]
lol I think I reviewed the wrong PR. Just pretend it's the one on mila-iqia/milatools

File: .github/workflows/python-publish.yml (hunk 1 above)
Comment (lebrice) [reply .github/workflows/python-publish.yml]: Fixed in ac45042

File: .github/workflows/build.yml (hunk 2 above)
Comment (lebrice) [reply .github/workflows/build.yml]: Fixed in 4700789

**Pull request closed without merging**